.gitignore

```diff
@@ -45,3 +45,4 @@ _build
 .idea
 .vscode
 *~
+.venv
```

Comment:
I'm rolling out a patch that will put this under `.env`, so would you mind doing that?  Thanks!